rel/haproxy.cfg

```diff
@@ -25,9 +25,9 @@ defaults
         option redispatch
         retries 4
         option http-server-close
-        timeout client 150000
-        timeout server 3600000
-        timeout connect 500
+        timeout client 60s
+        timeout server 60s
```

Comment:
That's a rather large change especially for the server timeout, a 60x reduction. Is it just to test code or it should be the new recommended timeout?

Reply:
this was just for testing, but rel/haproxy.cfg is not part of the deployment/install, is it? our installers only set up couchdb.

Reply:
At least for changes it's not needed as we already have a changes timeout max default of 60 seconds. That behavior is a bit odd, no matter what the user passes in as the timeout we always pick the minimum of the default and user's timeout: https://github.com/apache/couchdb/blob/e60e27554708f46a9e6528fa6049d025c1aba859/src/couch/src/couch_changes.erl#L401 I guess it's for cases of multi-tenant clusters to limit the timeout upper limit?